tools/wptrunner/wptrunner/browsers/sauce.py

```diff
@@ -44,10 +44,6 @@ def get_capabilities(**kwargs):
     tags = kwargs["sauce_tags"]
     tunnel_id = kwargs["sauce_tunnel_id"]
     prerun_script = {
-        "MicrosoftEdge": {
```

Comment:
I believe this is independent of the other Edge-related code, but I see no reason to keep this around, the reasoning from the RFC holds here too.